cleanify/cleanify.py

```diff
@@ -287,15 +287,15 @@ def _validate(self, epoch):
                         save_image(
                                 clean_image.cpu().data,
                                 f'{self._output_dir}/clean_e{epoch}_i{i}'
-                                + f'_{self._fx_name}.png'
+                                + f'_d{self._fx_name}.png'
                         )
 
                     # Save the last clean and dirty image pair into
                     # outputs directory at the end of the final epoch.
                     if epoch == (self._epochs - 1):
                         save_image(
                                 outputs.cpu().data,
-                                f'{self._output_dir}/cleaned_e{epoch}_i{i}.png'
+                                f'{self._output_dir}/cleaned_e{epoch}_i{i}._d{self._fx_name}.png'
```

Comment:
Minor thing: how about breaking at 79 chars
```
                        save_image(
                                outputs.cpu().data,
                                f'{self._output_dir}/cleaned_e{epoch}_i{i}.png'
                                + f'_d{self._fx_name}.png'
                        )
```
for consistency. There's also a `.` character before the `_d` in the currently proposed line but not in the other two `save_image()` calls.